introduction.md

```diff
@@ -37,11 +37,18 @@ research (including streaming replication, backups).
 - **Flexible management options** (compatible with existing PostgreSQL
 ecosystem and tooling).
 
-
 The rest of this section describes the design and motivation around the TimescaleDB
 architecture, including why time-series data is different, and how we leverage
-its characteristics when building TimescaleDB.
+its characteristics when building TimescaleDB. 
 
 **Next:** In part to understand TimescaleDB's design choices, let us ask: [What is time-series data?][time-series-data]
 
+# Download the Guide
```

Comment:
Can you make this a `##` h2 header instead of h1?

Also, it seems a little bare without an explanatory sentence.  Something like: 
"If you want a quick visual intro to TimescaleDB, click on the image below to download...", etc.
<img width="2005" alt="Screen Shot 2019-08-21 at 8 48 26 AM" src="https://user-images.githubusercontent.com/12288273/63433225-c9664180-c3f0-11e9-9ac8-208efbdcc309.png">

Reply:
done!